lib/client-pool.js

```diff
@@ -104,6 +104,7 @@ class Pool extends EventEmitter {
 
     if (this[kConnections] && this[kClients].length === this[kConnections]) {
       for (const { busy } of this[kClients]) {
+        /* istanbul ignore next */
```

Comment:
This should be possible to happen. Please remove ignore.

Reply:
Done. I leave it to you to add a test for it. I've tried with no luck.

Reply:
Please remove the ignore

Reply:
Update coming soon!

Reply:
Done!